test/apps/export-webpack/webpack.config.js

```diff
@@ -0,0 +1,73 @@
+const webpack = require('webpack');
```

Comment:
I copy-pasted this config file over from the only other webpack text project, `with-sourcemaps-webpack`.